scanner/src/components/Header/index.svelte

```diff
@@ -0,0 +1,34 @@
+<script lang="ts">
+  import LeftArrowSvg from "$assets/left-arrow.svg"
+  import QuestionSvg from "$assets/question.svg"
+  import MetaDataPng from "$assets/metadata.png"
+</script>
+
+<div class="wrapper">
+  <a href="/">
+    <img class="left-arrow" src={LeftArrowSvg} alt="" aria-hidden="true" />
+  </a>
+  <img class="question" src={MetaDataPng} alt="" aria-hidden="true" />
+  <a href="/">
+    <img class="metadata" src={QuestionSvg} alt="" aria-hidden="true" />
+  </a>
```

Comment:
Ask when it's unclear what something is supposed to do. This is not supposed to point to `/`. I asked, it is supposed to point to the documentation.